sklearn/tests/test_calibration.py

```diff
@@ -13,17 +14,189 @@
                                    assert_raises,
                                    ignore_warnings)
 from sklearn.datasets import make_classification, make_blobs
+from sklearn.linear_model import LogisticRegression
 from sklearn.naive_bayes import MultinomialNB
 from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
 from sklearn.svm import LinearSVC
 from sklearn.pipeline import Pipeline
 from sklearn.impute import SimpleImputer
-from sklearn.metrics import brier_score_loss, log_loss
-from sklearn.calibration import CalibratedClassifierCV
+from sklearn.metrics import brier_score_loss, log_loss, confusion_matrix,\
+    f1_score
+from sklearn.calibration import CalibratedClassifierCV, CutoffClassifier
+from sklearn.calibration import _get_binary_score
 from sklearn.calibration import _sigmoid_calibration, _SigmoidCalibration
 from sklearn.calibration import calibration_curve
 
 
+def test_cutoff_prefit():
+    calibration_samples = 200
+    X, y = make_classification(n_samples=1000, n_features=6, random_state=42,
+                               n_classes=2)
+
+    X_train, X_test, y_train, y_test = train_test_split(X, y,
+                                                        train_size=0.6,
+                                                        random_state=42)
+    lr = LogisticRegression().fit(X_train, y_train)
+
+    clf_roc = CutoffClassifier(lr, method='roc', cv='prefit').fit(
+        X_test[:calibration_samples], y_test[:calibration_samples]
+    )
+
+    y_pred = lr.predict(X_test[calibration_samples:])
+    y_pred_roc = clf_roc.predict(X_test[calibration_samples:])
+
+    tn, fp, fn, tp = confusion_matrix(
+        y_test[calibration_samples:], y_pred).ravel()
+    tn_roc, fp_roc, fn_roc, tp_roc = confusion_matrix(
+        y_test[calibration_samples:], y_pred_roc).ravel()
+
+    tpr = tp / (tp + fn)
+    tnr = tn / (tn + fp)
+
+    tpr_roc = tp_roc / (tp_roc + fn_roc)
+    tnr_roc = tn_roc / (tn_roc + fp_roc)
+
+    # check that the sum of tpr and tnr has improved
+    assert_greater(tpr_roc + tnr_roc, tpr + tnr)
+
+    clf_f1 = CutoffClassifier(
+        lr, method='f_beta', beta=1, cv='prefit', scoring='predict_proba').fit(
```

Comment:
can we also check that the beta != 1 case gets a different result, just to be sure that `beta` is being passed correctly?